spec/support/travel_advice_publisher_assertion_helpers.rb

```diff
@@ -0,0 +1,44 @@
+module TravelAdvicePublisherAssertionHelpers
+  def expect_creation_new_edition
```

Comment:
This should actually meant to be Create not creation. I was asserting that it landed on the correct page. I will change it to expect_create_new_edition.